operations/availabilityblocks.md

```diff
@@ -177,6 +178,7 @@ Returns all availability blocks filtered by services, unique identifiers and oth
 | `RateId` | string | required | Unique identifier of the [Rate](rates.md#rate) the block is assigned to. |
 | `VoucherId` | string | optional | Unique identifier of the [Voucher](vouchers.md#voucher) used to access specified private [Rate](rates.md#rate). |
 | `BookerId` | string | optional | Unique identifier of the [Customer](customers.md#customer) on whose behalf the block was made. |
+| `CompanyId` | string | optional | Unique identifier of the [Company](companies.md#company) of which the block is part of. |
```

Comment:
@MikeAdamsMews can you please come up with better wording? I quite like the version that's used for a `BookerId` not sure it's applicable for company as well.

Reply:
Maybe just "Unique identifier of the [Company](companies.md#company) linked to the block." That doesn't say anything about the nature of the relationship, simply that it is 'linked'.

Reply:
I've changed it to your suggestion @MikeAdamsMews. Thank you.